pkg/controller/registry/resolver/resolver.go

```diff
@@ -343,6 +346,30 @@ func (r *SatResolver) getBundleInstallables(preferredNamespace string, bundleSta
 			continue
 		}
 
+		for _, prop := range bundle.Properties {
+			if prop.Type != "olm.constraint" {
+				continue
+			}
+
+			var val struct {
+				Message string           `json:"message"`
+				Cel     *constraints.Cel `json:"cel"`
+			}
```

Comment:
Use the `Constraint` struct from API here?

Reply:
Yes. That's the intention after the api PR merged.